packages/react/src/OrderedList/OrderedList.tsx

```diff
@@ -10,17 +10,26 @@ import { OrderedListItem } from './OrderedListItem'
 
 export type OrderedListProps = {
   markers?: boolean
+  inverseColor?: boolean
 } & PropsWithChildren<OlHTMLAttributes<HTMLOListElement>>
 
 type OrderedListComponent = {
   Item: typeof OrderedListItem
 } & ForwardRefExoticComponent<OrderedListProps & RefAttributes<HTMLOListElement>>
 
 export const OrderedList = forwardRef(
-  ({ children, markers = true, className, ...restProps }: OrderedListProps, ref: ForwardedRef<HTMLOListElement>) => (
+  (
+    { children, markers = true, inverseColor, className, ...restProps }: OrderedListProps,
```

Comment:
Let’s sort lists of props alphabetically. This can make Git diffs less confusing (although dangling commas help there as well) and helps humans find specific props easier.

```suggestion
    { children, className, inverseColor, markers = true, ...restProps }: OrderedListProps,
```
```suggestion
    { children, markers = true, inverseColor, className, ...restProps }: OrderedListProps,
```